deis/azuredeploy.json

```diff
@@ -0,0 +1,225 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "newStorageAccountName": {
+            "type": "string",
+            "metadata": { 
+                 "description": "Name of the of the storage account for VM OS Disk"
+            } 
+        },
+        "vmNamePrefix": {
+            "type": "string",
+            "defaultValue": "coreos",
+            "metadata": { 
+                 "description": "Prefix for the names of the VMs"
+            } 
+        },
+        "virtualNetworkName": {
+            "type": "string",
+            "defaultValue": "myVNET",
+            "metadata": { 
+                 "description": "Name of the VNET hosting the VMs"
+            } 
+        },
+        "vmSize": {
+            "type": "string",
+            "defaultValue": "Standard_A1",
+            "metadata": { 
+                 "description": "Instance size for the VMs"
+            } 
+        },
+        "adminUserName": {
+            "type": "string",
+            "metadata": { 
+                 "description": "Username to login to the VMs"
+            } 
+        },
+        "sshKeyData": {
+            "type": "string",
+            "metadata": { 
+                 "description": "Public key for SSH authentication"
+            } 
+        },
+        "customData": {
+            "type": "string",
+            "metadata": { 
+                 "description": "Base64-encoded cloud-config.yaml file to deploy and start Fleet"
+            } 
+        },
+		"numberOfNodes":{
+			"type": "int",
+            "metadata": { 
+                 "description": "Number of member nodes"
+            },
+			"defaultValue": "3"
+		},
+		"imagePublisher": {
```

Comment:
Consider making these image related parameters into variables if this template takes a hard dependency on CoreOS. You could also have pub and offer hardocoded to CoreOS as variables and give the user a choice of SKU flavors as allowed values.